HousingSearchApi/V1/Infrastructure/Factories/ProcessesQueryGenerator.cs

```diff
@@ -25,15 +32,23 @@ public QueryContainer Create<TRequest>(TRequest request, QueryContainerDescripto
                 throw new ArgumentNullException($"{nameof(request).ToString()} shouldn't be null.");
             }
 
+
             _queryBuilder
                 .WithWildstarQuery(processListRequest.SearchText,
-                    new List<string> { "targetId", "targetType", "isOpen" })
+                   new List<string> { "patchAssignment.patchId" })
                 .WithExactQuery(processListRequest.SearchText,
-                    new List<string> { "targetId", "targetType", "isOpen" }, new ExactSearchQuerystringProcessor())
+                    new List<string> { "patchAssignment.patchId" }, new ExactSearchQuerystringProcessor())
                 .WithFilterQuery(processListRequest.TargetType, new List<string> { "targetType" })
-                .WithFilterQuery(processListRequest.TargetId.ToString(), new List<string> { "targetId" });
+                .WithFilterQuery(processListRequest.TargetId.ToString(), new List<string> { "targetId" })
```

Comment:
I am not sure if target id should be a filter query - it's a main search parameter for when retrieving processes on the person/property/tenure page and then the filters will be around date ranges, open status, etc